packages/augur-core/source/contracts/Augur.sol

```diff
@@ -48,11 +56,23 @@ contract Augur is IAugur {
     event MarketMigrated(address indexed market, address indexed originalUniverse, address indexed newUniverse);
     event UniverseForked(address indexed universe, IMarket forkingMarket);
     event UniverseCreated(address indexed parentUniverse, address indexed childUniverse, uint256[] payoutNumerators);
-    event OrderCanceled(address indexed universe, address indexed shareToken, address indexed sender, IMarket market, bytes32 orderId, Order.Types orderType, uint256 tokenRefund, uint256 sharesRefund);
-    event OrderPriceChanged(address indexed universe, bytes32 orderId, uint256 outcome, uint256 price);
-    // The ordering here is to match functions higher in the call chain to avoid stack depth issues
-    event OrderCreated(Order.Types orderType, uint256 amount, uint256 price, address indexed creator, bytes32 tradeGroupId, bytes32 orderId, IUniverse indexed universe, IMarket indexed market, ERC20Token kycToken, uint256 outcome);
-    event OrderFilled(address indexed universe, address filler, address creator, IMarket market, bytes32 orderId, uint256 price, uint256 outcome, uint256 fees, uint256 amountFilled, bytes32 tradeGroupId, bool orderIsCompletelyFilled, uint256 timestamp);
+
+    //  addressData
+    //  0:  kycToken
+    //  1:  orderCreator (Fill)
```

Comment:
Looks like orderCreator is also logged by logOrderCanceled, but it was removed from OrderCreated? Do we not need it for OrderCreated?

I suppose shareToken isn't really needed for OrderCanceled, so that's probably ok to remove.

Reply:
Ah that a mis-lable in the comment really. orderCreator will be on all OrderEvents

Reply:
Ah right, it is getting set in `getOrderDataForLogs`. Ok, never mind 👍